From: Jiri Slaby <jslaby@suse.cz>
Subject: Install presets to /usr/share
Patch-upstream: no

And not to home of a random (abuild) user.
---
 tools/realsense-viewer/CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/realsense-viewer/CMakeLists.txt
+++ b/tools/realsense-viewer/CMakeLists.txt
@@ -253,7 +253,7 @@ install(
 )
 #https://cmake.org/cmake/help/latest/command/install.html
 install(DIRECTORY presets/
-    DESTINATION $ENV{HOME}/Documents/librealsense2/presets
+    DESTINATION ${CMAKE_INSTALL_DATADIR}/librealsense2/presets
     FILES_MATCHING PATTERN "*.preset"
 )
 endif()
